resources/views/store/edit_registration.blade.php

```diff
@@ -57,22 +57,29 @@
                     <img src="{{ asset('store/assets/pregnancy-light.svg') }}" name="logo">
                     <h2>Children or pregnancy</h2>
                 </div>
+                @include('store.partials.add_child_form')
                 <div>
                     <table>
                         <thead>
                         <tr>
-                            <td>Age</td>
-                            <td>Month / Year</td>
-                            <td></td>
+                            <td class="age-col">Age</td>
+                            <td class="dob-col">Month / Year</td>
+                            @if ( in_array(auth::user()->centre->sponsor->shortcode, config('arc.verifies_children')) )
```

Comment:
This should work, but i can't get it to on my local. Something for karl to test with his browser scripts.